platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -151,6 +153,8 @@ public class MerkleStateRoot extends PartialNaryMerkleInternal
 
     private Metrics metrics;
 
+    private WritableRosterStore writableRosterStore;
```

Comment:
Why is this added here? It seems it should be looked up with a key in `WritableStates.getSingleton(String stateKey)`

Reply:
As per https://github.com/hashgraph/hedera-services/pull/15442/files#r1763534999
`RosterStore`s, like `PlatformStateStore`s, must be initialized before other stores in the `StoreFactory` are initialized.

I can't do `WritableStates.getSingleton(String stateKey)` as this is a store, not a state. The alternative to obtaining an instance of a store - to get it from the Factory - won't work here because the `StoreFactory` isn't initialized yet.

Reply:
Ok. Since it's outside of the norm, this should be explained in a javadoc.

Reply:
It's not outside of the norm in that the only two Stores on the "platform" side of the fence (`PlatformStateStore` and `RosterStateStore`) can only be obtained from this class. The 'regular' way of obtaining stores is via the StoreFactory but that's not even accessible to Platform, so no choice there.

That said, more docs won't hurt so I've added it.